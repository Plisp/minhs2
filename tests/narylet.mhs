main =
    let id x = x;
        f x y = x == y;
        x = f;
    in f 2;
