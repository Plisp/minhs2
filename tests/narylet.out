main :: (Int -> Bool) = let id :: forall a. (a -> a) x = x;
                            f :: (Int -> Int -> Bool) x y = (==) x y;
                            x :: (Int -> Int -> Bool) = f;
                        in f 2;
